contracts/payloads/IGP105/PayloadIGP105.sol

```diff
@@ -0,0 +1,585 @@
+                vault: wstUSR_USDC_USDT_VAULT,
+                vaultType: VAULT_TYPE.TYPE_2,
+                supplyToken: address(0), // Set at DEX level
+                borrowToken: USDT_ADDRESS,
+                baseWithdrawalLimitInUSD: 0,
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 20_000 // $20k
+            });
+            
+            setVaultLimits(VAULT_wstUSR_USDC_USDT);
+            VAULT_FACTORY.setVaultAuth(wstUSR_USDC_USDT_VAULT, TEAM_MULTISIG, true);
+        }
+        
+        {
+            address wstUSR_USDC_USDT_CONCENTRATED_VAULT = getVaultAddress(144);
+            
+            // [TYPE 2] WSTUSR-USDC<>USDC-USDT concentrated vault - Dust limits
+            VaultConfig memory VAULT_wstUSR_USDC_USDT_CONCENTRATED = VaultConfig({
+                vault: wstUSR_USDC_USDT_CONCENTRATED_VAULT,
+                vaultType: VAULT_TYPE.TYPE_2,
+                supplyToken: address(0), // Set at DEX level
+                borrowToken: USDT_ADDRESS,
+                baseWithdrawalLimitInUSD: 0,
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 20_000 // $20k
+            });
+            
+            setVaultLimits(VAULT_wstUSR_USDC_USDT_CONCENTRATED);
+            VAULT_FACTORY.setVaultAuth(wstUSR_USDC_USDT_CONCENTRATED_VAULT, TEAM_MULTISIG, true);
+        }
+    }
+
+    // @notice Action 5: Withdraw remaining $FLUID for Rewards
+    function action5() internal isActionSkippable(5) {
+        string[] memory targets = new string[](1);
+        bytes[] memory encodedSpells = new bytes[](1);
+
+        string
+            memory withdrawSignature = "withdraw(address,uint256,address,uint256,uint256)";
+
+        // Spell 1: Transfer FLUID to Team Multisig
+        {
+            uint256 FLUID_AMOUNT = 11_500 * 1e18;
+            targets[0] = "BASIC-A";
+            encodedSpells[0] = abi.encodeWithSignature(
+                withdrawSignature,
+                FLUID_ADDRESS,
+                FLUID_AMOUNT,
+                TEAM_MULTISIG,
+                0,
+                0
+            );
+        }
+
+        IDSAV2(TREASURY).cast(targets, encodedSpells, address(this));
+    }
+
+    // @notice Action 6: Update LBTC oracle and center price
+    function action6() internal isActionSkippable(6) {
+        {
+            address lBTC_cbBTC_DEX = getDexAddress(17);
+            IFluidDex(lBTC_cbBTC_DEX).updateCenterPriceAddress(
+                170,
+                1e4,
+                1 days
+            );
+        }
+        {
+            address CBBTC_WBTC_DEX_ADDRESS = getDexAddress(3);
+            IFluidDex(CBBTC_WBTC_DEX_ADDRESS).updateCenterPriceAddress(
+                171,
+                1e4,
+                1 days
+            );
+        }
+        {
+            address LBTC_cbBTC__cbBTC_VAULT = getVaultAddress(114);
+            IFluidVaultT1(LBTC_cbBTC__cbBTC_VAULT).updateOracle(
```

Comment:
smart vaults, get set oracle via nonce, so must set here e.g. 173 and use T2 interface  

same below